lib/constants/category.ts

```diff
@@ -0,0 +1,24 @@
+export type Category =
+  | 'ansible'
+  | 'batect'
+  | 'bazel'
+  | 'cd'
+  | 'ci'
+  | 'c'
+  | 'kubernetes'
+  | 'ruby'
+  | 'dotnet'
+  | 'rust'
+  | 'swift'
+  | 'php'
+  | 'jvm'
```

Comment:
I still have some doubts about java vs jvm. Sometimes we can be "a little bit technically wrong" with our naming because it's better/more expected by the user